curves/src/bls12_377/g1.rs

```diff
@@ -67,30 +67,38 @@ impl ShortWeierstrassParameters for Bls12_377G1Parameters {
     }
 }
 
+///
 /// G1_GENERATOR_X =
-/// 81937999373150964239938255573465948239988671502647976594219695644855304257327692006745978603320413799295628339695
+/// 180626689152567938232604559698399306213229526488826235533375072902470977089128486884883292831560455202598472059967
```

Comment:
Why has this changed?

Or, if considering changing, how about deriving it in a nothing up your sleeve manner? Like taking the minimal x value that gives a valid point?

(I see that it was like that and was removed! Curious why?)

Reply:
This is derived from the impl of `hash_to_curve` in this PR, with tests in `snarkvm-algorithms/hash_to_curve` confirming the new generator matches the input message.

As to why we changed this one, it's mainly to be generated uniformly with  BLS12-377 G2, BW6-761 G1 and G2, which didn't have a corresponding "increment from 0" generator.